Sources/GRPC/AsyncAwaitSupport/GRPCAsyncServerHandler.swift

```diff
@@ -270,7 +276,8 @@ internal final class AsyncServerHandler<
     self.compressionEnabledOnRPC = context.encoding.isEnabled
     self.compressResponsesIfPossible = true
     self.logger = context.logger
-    self.traceIDExtractor = context.traceIDExtractor
+    self.baggage = .topLevel // TODO: or carry from context?
```

Comment:
I suspect we'd want users to configure it (but default to `.topLevel`) and carry it over from context. There could be instance metadata or similar that the user provides.

Reply:
That aside, I think we should carry it from the context, I don't think there's any downside to it and gives the user control if they need it.